TYPO3.Flow/Classes/TYPO3/Flow/Resource/Target/FileSystemTarget.php

```diff
@@ -169,14 +175,14 @@ public function getName()
     /**
      * Publishes the whole collection to this target
      *
-     * @param \TYPO3\Flow\Resource\CollectionInterface $collection The collection to publish
+     * @param CollectionInterface $collection The collection to publish
      * @return void
      * @throws Exception
      */
     public function publishCollection(CollectionInterface $collection)
     {
         foreach ($collection->getObjects() as $object) {
-            /** @var \TYPO3\Flow\Resource\Storage\Object $object */
+            /** @var Object $object */
```

Comment:
`Object` is another candidate for ambiguous import name, though there is no "Object" class in PHP, but only the "object" type